edx_django_utils/logging/internal/filters.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django-based logging filters
+
+UserIdFilter: A logging.Filter that adds userid to the logging context
+
+RemoteIpFilter: A logging filter that adds the remote IP to the logging context
+"""
+
+from logging import Filter
+
+from crum import get_current_request, get_current_user
+
+
+class RemoteIpFilter(Filter):
+    """
+    Applies a filter to a record to extract the remote ip from the
+    request context.
```

Comment:
Can we use the comment you had above instead?
> A logging filter that adds the remote IP to the logging context.

The current docstring doesn't make as much sense to me.

Reply:
done.